src/segment-loader.js

```diff
@@ -1948,6 +1948,20 @@ export default class SegmentLoader extends videojs.EventTarget {
 
     segmentInfo.waitingOnAppends = 0;
 
+    // segments with no data
+    if (!segmentInfo.hasAppendedData_) {
+      // override settings for metadata only segments
+      segmentInfo.timestampOffset = segmentInfo.startOfSegment;
+      segmentInfo.timingInfo = {start: 0};
+      segmentInfo.waitingOnAppends++;
```

Comment:
We have to mock that we are waiting on an append, because in `checkAppendsDone` we do `segmentInfo. waitingOnAppends--` and then check against `0`.

Reply:
Instead of that, can this function just call `handleAppendsDone_` directly?

Reply:
We probably could but I was worried that we wouldn't get the abort handling that exists in `checkAppendsDone_`